include/structs/str_stat_index.h

```diff
@@ -0,0 +1,15 @@
+#ifndef GUARD_STR_STAT_INDEX_X
+#define GUARD_STR_STAT_INDEX_X
+
+#define STAT_INDEX_PHYSICAL 0 // Atk, Def
+#define STAT_INDEX_SPECIAL 1 // Sp Atk, Sp Def
+#define STAT_INDEX_ACCURACY STAT_INDEX_PHYSICAL
+#define STAT_INDEX_EVASION STAT_INDEX_SPECIAL
+
+// Needed to match in Blue/Sky. I wonder what was the reason for making it a struct as opposed to a simple int...
+struct StatIndex
+{
+    int id;
+};
```

Comment:
Do we want a `typedef` here like most of the other structs?

Reply:
Changed to `typedef`